CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 * [BUGFIX] Cortex: Fix GRPC stream clients not honoring overrides for call options. #5797
 * [BUGFIX] Ring DDB: Fix lifecycle for ring counting unhealthy pods as healthy. #5838
 * [BUGFIX] Ring DDB: Fix region assignment. #5842
+* [CHANGE] Upgrade Dockerfile Node version from 14x to 18x.
```

Comment:
Can you please move this under `master / unreleased` section? Also please mention the PR number